meta-lmp-base/classes/kernel-lmp-fitimage.bbclass

```diff
@@ -278,7 +300,7 @@ fitimage_emit_section_config() {
 	if [ -n "${dtb_image}" ]; then
```

Comment:
this sounds like it should be `dtb_image_sect` as it will be the value used above on the fdt_line var

Reply:
In the end dtb_image_sect depends on dtb_image, and why it should work.